R/Report_FlagOverTime.R

```diff
@@ -3,9 +3,8 @@
 #' Create a table of longitudinal study data by site, study, or country, showing
 #' flags over time.
 #'
+#' @inheritParams shared-params
```

Comment:
This is great! so the idea is that _all_ params in the `shared-params` header would be popped into the given function's header? in which case, we'd probably want a series of `shared-params` objects for different common combinations of params. or is it possible to selectively include them by specifying the name of the param?

Reply:
When you @inheritParams, roxygen automatically only grabs the params that are used in the function. So that one object can include *all* params, and roxygen will just grab the ones that apply.

Reply:
oh nice!